core/src/main/java/hudson/init/InitMilestone.java

```diff
@@ -100,7 +100,7 @@ public static TaskBuilder ordering() {
         TaskGraphBuilder b = new TaskGraphBuilder();
         InitMilestone[] v = values();
         for (int i=0; i<v.length-1; i++)
-            b.add(null, Executable.NOOP).requires(v[i]).attains(v[i+1]);
+            b.add("ORDER: " + v[i] + " -> " + v[i+1], Executable.NOOP).requires(v[i]).attains(v[i+1]);
```

Comment:
@tfennelly no, it's just for debugging/logging. there's no other way to get any info about the milestones than their title, which was returning 'null' for these